### PR TITLE
Update minimal requirements for implementation

### PR DESCRIPTION
**Pull request by sfyz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ The BYOLLM Open Connector is a commitment to community-driven growth and innovat
 
 1. Clone this repository.
 2. Implement an HTTP REST service using the [LLM Open Connector OpenAPI specification](api-specs/llm-open-connector/llm-open-connector.yml).
```
Comment (sfyz) [inline README.md:14]:
```suggestion
2. Implement an HTTP REST service using the [LLM Open Connector OpenAPI specification](api-specs/llm-open-connector/llm-open-connector.yml). This specification contains three endpoints: `/completions`, `/chat/completions`, and `/embeddings`.
```

File: README.md [hunk 2]
```diff
@@ -11,8 +11,10 @@ The BYOLLM Open Connector is a commitment to community-driven growth and innovat
 ### Usage
 
 1. Clone this repository.
-2. Implement an HTTP REST service using the [LLM Open Connector OpenAPI specification](api-specs/llm-open-connector/llm-open-connector.yml).
-3. Test your service connection using Bring Your Own Large Language Model (BYOLLM) in Einstein 1 Studio.
+2. Implement an HTTP REST service using the [LLM Open Connector OpenAPI specification](api-specs/llm-open-connector/llm-open-connector.yml). This specification contains three endpoints: `/completions`, `/chat/completions`, and `/embeddings`.
+   - The `/completions` endpoint is required for Einstein Studio.
+   - The `/chat/completions` endpoint is required for Prompt Builder.
```
Comment (sfyz) [inline README.md:16]:
```suggestion
   - The `/chat/completions` endpoint is required for Prompt Builder.
   - The `/embeddings` endpoint is only needed for embeddings use cases.
```

File: README.md [hunk 3]
```diff
@@ -11,8 +11,11 @@ The BYOLLM Open Connector is a commitment to community-driven growth and innovat
...
```
Comment (sfyz) [inline README.md:16]:
```suggestion
   - The `/chat/completions` endpoint is required for Prompt Builder and chat-based use cases.
```

File: README.md [hunk 4]
```diff
@@ -11,8 +11,11 @@ The BYOLLM Open Connector is a commitment to community-driven growth and innovat
 ### Usage
 
 1. Clone this repository.
-2. Implement an HTTP REST service using the [LLM Open Connector OpenAPI specification](api-specs/llm-open-connector/llm-open-connector.yml).
-3. Test your service connection using Bring Your Own Large Language Model (BYOLLM) in Einstein 1 Studio.
+2. Implement an HTTP REST service using the [LLM Open Connector OpenAPI specification](api-specs/llm-open-connector/llm-open-connector.yml). This specification contains three endpoints: `/completions`, `/chat/completions`, and `/embeddings`.
+   - The `/completions` endpoint is required for Einstein Studio.
```
Comment (sfyz) [inline README.md:15]:
```suggestion
   - The `/completions` endpoint is required for Einstein Studio and simple text generation use cases.
```

File: README.md [hunk 5]
```diff
@@ -11,8 +11,11 @@ The BYOLLM Open Connector is a commitment to community-driven growth and innovat
 ### Usage
 
 1. Clone this repository.
-2. Implement an HTTP REST service using the [LLM Open Connector OpenAPI specification](api-specs/llm-open-connector/llm-open-connector.yml).
-3. Test your service connection using Bring Your Own Large Language Model (BYOLLM) in Einstein 1 Studio.
+2. Implement an HTTP REST service using the [LLM Open Connector OpenAPI specification](api-specs/llm-open-connector/llm-open-connector.yml). This specification contains three endpoints: `/completions`, `/chat/completions`, and `/embeddings`.
+   - The `/completions` endpoint is required for Einstein Studio and simple text generation use cases.
```
Comment (sfyz) [inline README.md:15]:
```suggestion
   - The `/completions` endpoint is used for simple text generation use cases. It is required for Einstein Studio.
```

File: README.md [hunk 6]
```diff
@@ -11,8 +11,11 @@ The BYOLLM Open Connector is a commitment to community-driven growth and innovat
 ### Usage
 
 1. Clone this repository.
-2. Implement an HTTP REST service using the [LLM Open Connector OpenAPI specification](api-specs/llm-open-connector/llm-open-connector.yml).
-3. Test your service connection using Bring Your Own Large Language Model (BYOLLM) in Einstein 1 Studio.
+2. Implement an HTTP REST service using the [LLM Open Connector OpenAPI specification](api-specs/llm-open-connector/llm-open-connector.yml). This specification contains three endpoints: `/completions`, `/chat/completions`, and `/embeddings`.
+   - The `/completions` endpoint is used for simple text generation use cases. It is required for Einstein Studio.
+   - The `/chat/completions` endpoint is required for Prompt Builder and chat-based use cases.
```
Comment (sfyz) [inline README.md:16]:
```suggestion
   - The `/chat/completions` endpoint is used for chat-based use cases. It is required for Prompt Builder.
```

File: README.md [hunk 7]
```diff
@@ -11,8 +11,11 @@ The BYOLLM Open Connector is a commitment to community-driven growth and innovat
 ### Usage
 
 1. Clone this repository.
-2. Implement an HTTP REST service using the [LLM Open Connector OpenAPI specification](api-specs/llm-open-connector/llm-open-connector.yml).
-3. Test your service connection using Bring Your Own Large Language Model (BYOLLM) in Einstein 1 Studio.
+2. Implement an HTTP REST service using the [LLM Open Connector OpenAPI specification](api-specs/llm-open-connector/llm-open-connector.yml). This specification contains three endpoints: `/completions`, `/chat/completions`, and `/embeddings`.
+   - The `/completions` endpoint is used for simple text generation use cases. It is required for Einstein Studio.
+   - The `/chat/completions` endpoint is used for chat-based use cases. It is required for Prompt Builder.
+   - The `/embeddings` endpoint is only needed for embeddings use cases.
```
Comment (sfyz) [inline README.md:17]:
```suggestion
   - The `/embeddings` endpoint is used for embeddings use cases. It is optional.
```

File: README.md [hunk 8]
```diff
@@ -11,8 +11,11 @@ The BYOLLM Open Connector is a commitment to community-driven growth and innovat
 ### Usage
 
 1. Clone this repository.
-2. Implement an HTTP REST service using the [LLM Open Connector OpenAPI specification](api-specs/llm-open-connector/llm-open-connector.yml).
-3. Test your service connection using Bring Your Own Large Language Model (BYOLLM) in Einstein 1 Studio.
+2. Implement an HTTP REST service using the [LLM Open Connector OpenAPI specification](api-specs/llm-open-connector/llm-open-connector.yml). This specification contains three endpoints: `/completions`, `/chat/completions`, and `/embeddings`.
```
Comment (rsexton404) [inline README.md:14]:
```suggestion
2. Implement an HTTP REST service using the [LLM Open Connector OpenAPI specification](api-specs/llm-open-connector/llm-open-connector.yml). This service can contain three endpoints: `/completions`, `/chat/completions`, and `/embeddings`.
```

Comment (rsexton404) [reply README.md:14]: What do you think, @sfyz?

File: README.md [hunk 9]
```diff
@@ -11,8 +11,11 @@ The BYOLLM Open Connector is a commitment to community-driven growth and innovat
...
+   - The `/completions` endpoint is used for simple text generation use cases. It is required for Einstein Studio.
+   - The `/chat/completions` endpoint is used for chat-based use cases. It is required for Prompt Builder.
```
Comment (sfyz) [inline README.md:16]:
```suggestion
   - The `/chat/completions` endpoint is used for chat-based use cases. It is required for Prompt Builder and Agentforce.
```

File: README.md (hunk 8 above)
Comment (sfyz) [reply README.md:14]: great! thanks!

**Pull request merged**